packages/docs/docs/getting-started/upgrade-to-v4.md

```diff
@@ -439,6 +439,106 @@ See the [`session` option docs](https://next-auth.js.org/configuration/options#s
 
 Introduced in https://github.com/nextauthjs/next-auth/pull/3144
 
+## Database migration
+
+NextAuth.js v4 has a slightly different database schema compared to v3. If you're using any of our adapters and want to upgrade, you can use on of the below schemas.
+
+They are designed to be run directly against the database itself. So instead of having one in Prisma syntax, one in TypeORM syntax, etc. we've decided to just make one for each underlying database type. i.e. one for Postgres, one for MySQL, one for MongoDB, etc.
+
+#### MySQL
+
+```sql
+/* ACCOUNT */
+ALTER TABLE accounts
+CHANGE "access_token_expires" "expires_at" int
+CHANGE "user_id" "userId" varchar(191)
+ADD CONSTRAINT fk_user_id FOREIGN KEY (user_id) REFERENCES users(id)
+RENAME COLUMN "provider_id" "provider"
+RENAME COLUMN "provider_account_id" "providerAccountId"
+DROP COLUMN "provider_type"
+DROP COLUMN "compound_id"
+DROP COLUMN "created_at"
+DROP COLUMN "updated_at"
```

Comment:
Let's comment that these have never been necessary for NextAuth.js to work, hence they are dropped, but can be kept if they want to.